Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -4150,12 +4193,14 @@ def clone_appsec_config_version_command(client: Client,
                                         config_id: str,
                                         create_from_version: str,
                                         do_not_clone: str,
+                                        rule_update: bool = True,
```

Comment:
Please set default as false so it won't affect other users

Reply:
def clone_appsec_config_version(self,
                                config_id: str,
                                create_from_version: str) -> Dict:
    """
    Create a new version of security configuration from a previous version
    Args:
        config_id
        versionId

    Returns:
        <Response [204]>
    """
    body = {
        "createFromVersion": int(create_from_version),
        "ruleUpdate": True
    }